test/ex_machina_test.exs

```diff
@@ -1,6 +1,10 @@
 defmodule ExMachinaTest do
   use ExUnit.Case
 
+  defmodule FooBar do
```

Comment:
This is part of a different commit, but the change is so small and we use it in this PR's tests, so I included it in this PR. This is the commit https://github.com/thoughtbot/ex_machina/pull/408/commits/648117cd48b7b3acd22cbae2b5a0e322a4115397